src/gui/qgsmetadatawidget.h

```diff
@@ -43,16 +43,50 @@ class GUI_EXPORT QgsMetadataWidget : public QWidget, private Ui::QgsMetadataWidg
 
     /**
      * Constructor for the wizard.
+     * \note
+     * For use with Layer. Sets mMetadata if pointer is valid
+     *  calls setMetadata, using mMetadata
+     * \param layer to set the main QgsLayerMetadata with mLayer->metadata() when not nullptr
+     * \see mMetadata
+     * \see setMetadata
+     * \since QGIS 3.0
      */
     QgsMetadataWidget( QWidget *parent, QgsMapLayer *layer = nullptr );
 
+    /**
+     * Set a QgsLayerMetadata object.
```

Comment:
If you change this to "Sets the \a metadata to display in the widget", you can drop the unnecessary "\param" line.

Reply:
done